modules/@apostrophecms/modal/ui/apos/components/AposModal.vue

```diff
@@ -126,13 +126,13 @@ export default {
     },
     gridModifier() {
       if (this.hasLeftRail && this.hasRightRail) {
-        return 'apos-modal__main--grid apos-modal__main--with-rails';
+        return 'apos-modal__main--with-rails';
```

Comment:
`.apos-modal__main` is now display: grid all the time. It children being able to fill empty modal space lets v-tooltip get positioned correctly. Not a great reason for this change but =/